docs/user_guide/fairness_in_machine_learning.rst

```diff
@@ -39,6 +39,79 @@ The Fairlearn package is most applicable to two kinds of harms:
   face recognition, document search, or product recommendation.
 
+Concept Glossary
+----------------------------
+
+The concepts outlined in this glossary are relevant to sociotechnical contexts. 
+
+Construct Validity
+^^^^^^^^^^^^^^^^^^
+
+We explore the concept of construct validity, as described in in Measurement and Fairness [#4]_ , and as it relates to sociotechnical contexts. This in intended to illuminate the concept to readers who are working on solutions in sociotechnical contexts and may also be referenced in explorations that detail how to apply the concept to specific scenarios.
+
+Key Terms 
+~~~~~~~~~
+
+- **Sociotechnical context** – A context in which both societal and technical aspects shape who might be harmed by AI systems and how (see fairlearn.ai for more detailed definition).
+
+- **Unobservable theoretical construct** – An idea or concept that is unobservable and cannot be directly measured but must instead be inferred through observable measurements defined in a measurement model. 
+
+- **Measurement model** – The method and approach used to measure the unobservable theoretical construct.
+
+- **Construct reliability** – This is closely related to construct validity (detailed below) and can be thought of as the extent to which the measurements of an unobservable theoretical construct remain the same when measured at different points in time. A lack of construct reliability can either be due to a misalignment between the understanding of the unobservable theoretical construct and the methods being used to measure that construct, or to changes to the construct itself.
+
+- **Construct validity** – This can be thought of as the extent to which the measurement model can be used to produce meaningful and useful results.
+
+Key Term Examples  - Unobservable theoretical constructs and Measurement models
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- **Fairness** is an example of an unobservable theoretical construct. Several measurement models exist for measuring fairness, including demographic parity. See Fairlearn Metrics for more examples of measurement models for measuring fairness. 
+
+- **Teacher effectiveness** is an example of an unobservable theoretical construct. Common measurement models include student performance on standardized exams and qualitative feedback for the teacher’s students.
+
+- **Socioeconomic status** is an example of an unobservable theoretical construct. A common measurement model includes annual household income. 
+
+- **Patient benefit** is an example of an unobservable theoretical construct. A common measurement model involves patient care costs.
+
+**Note:** We cite several examples of unobservable theoretical constructs and measurement models for the purpose of explaining the key terms outlined above.  Please reference Measurement and Fairness for more detailed examples.
+ 
+What is Construct Validity? 
```

Comment:
More of a high-level question: Should this live on this page, or on a sub-page dedicated to this topic? If it stays on this page, is this the place where it should go, or rather at the end of what already exists? @fairlearn/fairlearn-maintainers

Reply:
What would be the rationale for moving it outside the user guide? Because there isn't currently functionality to create tests for these, for instance?

I could see an argument to it being in Section 5 on further resources (under a subheader on e.g., "Related Fairness Concepts" or something)

Reply:
No, I meant within the user guide, but in a separate section (or separate subsection of section 1). It should definitely not be outside the user guide.

Reply:
What is the resolution here? 
Same question would apply to the other pending concept definition PRs (i.e: Abstraction Traps, what is the location there)? @romanlutz  @mmadaio

Reply:
There's a PR open for the abstraction traps for the user guide (for now)!

https://github.com/fairlearn/fairlearn/pull/809/files